src/PDFDoc.php

```diff
@@ -30,6 +30,9 @@
 use ddn\sapp\pdfvalue\PDFValueSimple;
 use ddn\sapp\pdfvalue\PDFValueHexString;
 use ddn\sapp\pdfvalue\PDFValueString;
+use ddn\sapp\helpers\cms;
+use ddn\sapp\helpers\x509;
+use ddn\sapp\helpers\asn1;
```

Comment:
Why those clases are inside `CMS.php`, shouldn't they be in their own file?

Also the file is `CMS.php`, so, the class name must be `CMS`  too(_on uppercase_), **PSR-4** is case sensitive

Reply:
sorry not familiar with namespace :) will be updated later.
feel free to suggest the changes to those adjustment.

Reply:
I think maintaining **PSR-4** is an essential adjustment

Reply:
so am i just have to move classes to own file named same as class name and add `namespace ddn\sapp\helpers;` in file header or still any other requirements?

Reply:
`class CMS {` great

> just have to move classes to own file named same as class name and add namespace ddn\sapp\helpers; in file header or still any other requirements?

Yes, it wouldn't need any other changes